client/src/components/add-job-post-component/application-form-fields/index.jsx

```diff
@@ -0,0 +1,143 @@
+          <H3>Application form fields</H3>
+        </div>
+
+        {/* Name Field */}
+        <TextField
+          disabled
+          fullWidth
+          className={styles.dotted}
+          sx={{ m: 1, width: "47%" }}
+          label="Name"
+          helperText="*Required"
+          required
+          InputLabelProps={{ required: false }}
+        />
+
+        {/* Email Field */}
+        <TextField
+          disabled
+          fullWidth
+          className={styles.dotted}
+          sx={{ m: 1, width: "47%" }}
+          label="Email Address"
+          helperText="*Required"
+          required
+          InputLabelProps={{ required: false }}
+        />
+
+        {/* Phone Field */}
+        <TextField
+          disabled
+          fullWidth
+          className={styles.dotted}
+          sx={{ m: 1, width: "47%" }}
+          label="Phone"
+          helperText="*Required"
+          required
+          InputLabelProps={{ required: false }}
+        />
+
+        {/* Postal Code Field */}
+        <TextField
+          disabled
+          fullWidth
+          className={styles.dotted}
+          sx={{ m: 1, width: "47%" }}
+          label="Postal Code"
+          helperText="*Required"
+          requiredf
+          InputLabelProps={{ required: false }}
+        />
+
+        {/* Status In Canada Field */}
+        <FormControl disabled fullWidth sx={{ m: 1, width: "96%" }}>
+          <InputLabel>Status In Canada</InputLabel>
+          <Select className={styles.dotted} required />
+          <FormHelperText>*Required</FormHelperText>
+        </FormControl>
+
+        {/* Cover Letter Field */}
+        <TextField
+          disabled
+          fullWidth
+          className={styles.dotted}
+          sx={{ m: 1, width: "96%" }}
+          label="Cover Letter"
+          multiline
+          rows={4}
+          InputLabelProps={{ shrink: true, required: false }}
+          required
+        />
+
+        {/* Dummy resume upload Field */}
+        <Box sx={{ margin: "8px" }}>
+          <Typography color="#9E9E9E" pb="13px">
+            *Upload Resume
+          </Typography>
+          <Container
+            sx={{
+              border: "1px dashed #E0E0E0",
+              borderRadius: "15px",
+              color: "#AAAFBA",
+              display: "flex",
+              justifyContent: "center",
+              alignItems: "center",
+              flexDirection: "column",
+              padding: "30px 0",
+            }}
+          >
+            <Container sx={{ width: "auto" }}>
+              <IconButton sx={{ backgroundColor: "#E6ECFB" }}>
+                <UploadFileIcon />
+              </IconButton>
+            </Container>
+            <Typography variant="body1" pb="10px" pt="10px">
+              <span className={styles.underline}>Click to upload</span>
+              &nbsp;or drag and drop
+            </Typography>
+            <Typography variant="body2">
+              SVG, PNG, JPG or GIF (max. 3MB)
```

Comment:
Maybe PDFs would be better for the resume upload but I'm not sure what our stakeholders want.

Reply:
We are going with PDF uploads for now, as that is what TCES wants!